django_project/base/views/project.py

```diff
@@ -236,3 +241,102 @@ def get_redirect_url(self, slug):
         project.approved = True
         project.save()
         return reverse(self.pattern_name)
+
+
+def paymentview(request):
+    '''
+    :param request:
+    :return:
+    To do
+    1. Try catch blocks
+    2. Link to javascript form
+    3. Add webhooks
+    '''
+    stripe.api_key = "sk_test_OHW7bvLJhDtm1k2pI8AwIiEY"
+    ourclientid = "ca_BM5OIMXOFqSOBjJcr4DrGHTsK3tLFuW3"
+    myaccount = stripe.Account.retrieve("acct_1AzNDQGz66mVbJVl")
+    myaccountid = myaccount.id
+    '''
+        Stripe connect
+    '''
+    returnid = request.GET.get(
+        'code')  # token that is sent to retrieve client id to be stored in db
+    # POST request to OAUTH
+
+    r = requests.post(
+        "https://connect.stripe.com/oauth/token",
+        data={
+            'client_secret': stripe.api_key,
+            'code': returnid,
+            'grant_type': "authorization_code"})
+    temp = json.loads(r.content)
+    if r.status_code == 200:
+        merchantid = temp['stripe_user_id']
+    # Hard coded clientid returned- acct_1B1rtDBWHazZ6NrT
+    else:
+        merchantid = "acct_1B1rtDBWHazZ6NrT"
+
+    # 1. View account balances of platform and connected account and card
+    merchantbalance = stripe.Balance.retrieve(
+        stripe_account=merchantid
+    )
+    # clientamount = clientbalance.amount
+    platformbalance = stripe.Balance.retrieve(
+        stripe_account=myaccountid
+    )
+    # platformamount = platformbalance.amount
+    # 3. Customer account info
+    custtok = stripe.Token.create(
+        bank_account={
+            "country": 'US',
+            "currency": 'usd',
+            "account_holder_name": 'Avery White',
+            "account_holder_type": 'individual',
+            "routing_number": '110000000',
+            "account_number": '000123456789'
```

Comment:
i think this should probably also live in the settings

Reply:
And settings should read it from an environment variable probably....